src/settings.lua

```diff
@@ -3,4 +3,5 @@ return {
   chunkSize = 1000,
   scenesDir = "/res/scenes/",
   saveDir = "saves/",
+  logLevel = "INFO",
```

Comment:
The default log level for users is INFO. If you run the game with `./run`, debug mode is still turned on by default.